lib/optimizely.rb

```diff
@@ -258,6 +258,8 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
         }, @logger, Logger::ERROR
       )
 
+      return false unless user_inputs_valid?(attributes)
```

Comment:
Can we combine `Optimizely::Helpers::Validator.inputs_valid?` with `user_inputs_valid?`? Seems like we could just put all user inputs into one dictionary and have them all validated

Reply:
Combined validations: https://github.com/optimizely/ruby-sdk/compare/rashid/input-validations...rashid/input-validations-2
It seems a review-able change plz confirm to continue with this PR?